examples/notebooks/beam-ml/run_inference_tensorflow.ipynb

```diff
@@ -414,6 +413,8 @@
         "'x': tf.io.FixedLenFeature([], tf.float32),\n",
         "}\n",
         "\n",
+        "# tf.function compiles the function into a callable TF graph.\n",
+        "# RunInference relies on calling a TF graph as a model.\n",
```

Comment:
IIRC it explicitly requires an input signature with dtype string - https://github.com/tensorflow/tfx-bsl/blob/49baf317527cc53d2dfb7d6c37b59757cd6d83a8/tfx_bsl/beam/run_inference.py#L627 - do you know why that is? Regardless, could you call that out?

Reply:
@ryanthompson591 reminder about this lingering comment. Once that is in we can merge